ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -840,7 +840,7 @@ tests:
     cluster_profile: vsphere-elastic
     workflow: openshift-e2e-vsphere-staticip
   timeout: 6h0m0s
-- as: e2e-vsphere-ovn-multi-network
+- as: e2e-vsphere-ovn-multi-network-tech-preview
```

Comment:
Standard here is [techpreview](https://github.com/openshift/sippy/blob/main/pkg/variantregistry/ocp.go#L483) like [e2e-vsphere-ovn-techpreview](https://github.com/openshift/release/blob/635a759b632a6adb44f33c71f1d0b1581b0e1031/ci-operator/config/openshift/installer/openshift-installer-main.yaml#L818)